### PR TITLE
Move file via uploader

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
When sharing a file with owncloud it is now possible to choose between copy and move, just like in the upload from within our app.
But in despite of this the default is always copy (as e.g. no one wants to move whatsapp images to owncloud as this might break the app).
Of course it can happen that a file is moved during sharing that cannot be moved (e.g. a cached image), then it will just copy it and fails silently.

@AndyScherzinger can you help me with the layout? I tried to do the same as in the upload_files_layout.xml, but it does not really work :/

wronglayout

### CODE REVIEW

File: res/layout/uploader_layout.xml [hunk 1]
```diff
@@ -17,24 +17,17 @@
...
-
-	<FrameLayout android:layout_height="fill_parent"
-		android:layout_width="fill_parent"
-		android:id="@+id/frameLayout1"
-		android:layout_below="@+id/drawer_username"
-		android:layout_above="@+id/linearLayout1">
+	<FrameLayout
+		android:layout_height="388dp"
+		android:layout_width="match_parent"
+		android:id="@+id/frameLayout1">
```
Comment (przybylski) [inline res/layout/uploader_layout.xml]: If you are never accessing this layout in code then you should remove this id.

File: res/layout/uploader_layout.xml [hunk 2]
```diff
@@ -43,33 +36,75 @@
 			android:dividerHeight="1dip">
 		</ListView>
 
+		<TextView android:layout_width="fill_parent"
+            android:text="@string/uploader_top_message"
+            android:layout_height="wrap_content"
+            android:id="@+id/drawer_username"
+            android:textColor="@android:color/black"
```
Comment (przybylski) [inline res/layout/uploader_layout.xml]: Isn't there a color/owncloudDarkTextColor (or something similar) resource in the project?

File: res/layout/uploader_layout.xml [hunk 3]
```diff
@@ -43,33 +36,75 @@
...
 	<LinearLayout
-	    android:id="@+id/linearLayout1"
-	    android:layout_width="fill_parent"
-	    android:layout_height="wrap_content"
-	    android:layout_alignParentBottom="true"
-	    android:orientation="horizontal" >
+		android:orientation="horizontal"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:paddingTop="8dp"
```
Comment (przybylski) [inline res/layout/uploader_layout.xml]: There are standard padding in metrics files

File: res/layout/uploader_layout.xml [hunk 4]
```diff
@@ -43,33 +36,75 @@
...
+		android:paddingLeft="16dp"
+		android:paddingRight="16dp">
+
+		<RadioGroup xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (przybylski) [inline res/layout/uploader_layout.xml]: This namespace is already defined at the root node

File: src/com/owncloud/android/ui/activity/Uploader.java [hunk 5]
```diff
@@ -424,6 +428,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     private void populateDirectoryList() {
         setContentView(R.layout.uploader_layout);
+
+        mRadioBtnMoveFiles = (RadioButton) findViewById(R.id.uploader_radio_move);
+        mRadioBtnCopyFiles = (RadioButton) findViewById(R.id.uploader_radio_copy);
+        mRadioBtnCopyFiles.setChecked(true);
```
Comment (przybylski) [inline src/com/owncloud/android/ui/activity/Uploader.java]: Doesn't it override user decision when user de-selects this radio button and rotate the phone?

**Comment by przybylski** [comment]
Review done

**Comment by przybylski** [comment]
@tobiasKaminsky about your layout problems. You need to make the weight of file list 1 so it would stretch to the whole screen. 
As a design i would remove "choose upload folder:" view and move that to actionbar title.
Since this is triggered via system and chooser user already knows what app is he in, We don't need to take precious space and announce everywhere that he is in ownCloud (he knows that :) but that just my two ¢)

File: res/layout/uploader_layout.xml (hunk 2 above)
Comment (tobiasKaminsky) [inline res/layout/uploader_layout.xml]: There is textColor, which is also black. So I am using this.

File: res/layout/uploader_layout.xml (hunk 3 above)
Comment (tobiasKaminsky) [inline res/layout/uploader_layout.xml]: I am not that good with layouting. What do you mean?
Maybe also @AndyScherzinger can do this part.

File: src/com/owncloud/android/ui/activity/Uploader.java (hunk 5 above)
Comment (tobiasKaminsky) [inline src/com/owncloud/android/ui/activity/Uploader.java]: No, but it will overwrite it again when changing a directory.
So I have to fix this.

**Comment by tobiasKaminsky** [comment]
Thanks for the hint with weight=1.
Regarding the title and remaining layout: I left everything unchanged.
But it is a really good idea to change this.
Is this ok for you, @jancborchardt?

**Comment by jancborchardt** [comment]
Yeah, @przybylski’s suggestion sounds good. :)

**Comment by jesmrec** [comment]
Native uploader available. Closing this,

**Pull request closed without merging**